### PR TITLE
[FIX][15.0] mail_debrand: test_mail_digest update for 15.0

### PR DESCRIPTION
**Pull request by dhx**
With 15.0 test_mail_digest seems to be broken:

```
ERROR: TestMailDebrandDigest.test_mail_digest
Traceback (most recent call last):
  File "/data/build/addons-oca/addons/mail_debrand/tests/test_mail_debrand_digest.py", line 51, in test_mail_digest
    "kpi_data": self.mail_digest_id.compute_kpis(
AttributeError: 'digest.digest' object has no attribute 'compute_kpis'
```

This PR solves the issue for us by:
- updating function calls to use new internal names
- using qweb_view engine to be able to specify the template via xmlid (the qweb engine doesn't seem to accept xmlids anymore since  https://github.com/odoo/odoo/commit/69dfaf4ca5a17f880264b2d95f782c7136613e95 )

### CODE REVIEW

**Review by MiquelRForgeFlow — APPROVED** [review]

**Comment by dhx** [comment]
Hi @pedrobaeza any chance you can have a short look?

**Comment by pedrobaeza** [comment]
I'm not updated with the changes done on this version. Please ask the contributors of such updates, and maybe I should be removed from the maintainers.

**Comment by dhx** [comment]
> I'm not updated with the changes done on this version. Please ask the contributors of such updates, and maybe I should be removed from the maintainers.

Sure, @pedrobaeza thanks anyway for your response.

Not sure who is best to ask, but as you made the last commit in the 15.0 branch on this module @yajo maybe you can give a second review?

**Comment by yajo** [comment]
Hi there @dhx! Thanks for your contribution 😊 

I'm having a review and there's something weird here. Looking at the logs of the last test execution in the main branch, I can see [this line](https://github.com/OCA/social/runs/7200211052?check_suite_focus=true#step:8:672):

    2022-07-05 16:19:13,258 259 INFO odoo odoo.addons.mail_debrand.tests.test_mail_debrand_digest: Starting TestMailDebrandDigest.test_mail_digest ... 

It seems to indicate that the test you were trying to fix was working fine.

However I can also see [these other lines](https://github.com/OCA/social/runs/7200211052?check_suite_focus=true#step:8:655) just above that one:

```
2022-07-05 16:19:13,250 259 INFO odoo odoo.service.server: Starting post tests 
2022-07-05 16:19:13,253 259 ERROR odoo odoo.tests.loader: Can not `import mail_autosubscribe`. 
Traceback (most recent call last):
  File "/opt/odoo/odoo/tests/loader.py", line 33, in _get_tests_modules
    mod = importlib.import_module('.tests', modpath)
  File "/usr/lib/python3.8/importlib/__init__.py", line 127, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "<frozen importlib._bootstrap>", line 1014, in _gcd_import
  File "<frozen importlib._bootstrap>", line 991, in _find_and_load
  File "<frozen importlib._bootstrap>", line 975, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 671, in _load_unlocked
  File "<frozen importlib._bootstrap_external>", line 848, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/__w/social/social/mail_autosubscribe/tests/__init__.py", line 1, in <module>
    from . import test_mail_autosubscribe
  File "/__w/social/social/mail_autosubscribe/tests/test_mail_autosubscribe.py", line 5, in <module>
    from odoo_test_helper import FakeModelLoader
ModuleNotFoundError: No module named 'odoo_test_helper'
```

I guess that because of the import error, we're not getting the correct failure in these tests.

I opened https://github.com/OCA/social/pull/932 so we can see that effect. That should make this PR more obviously needed.

**Comment by dhx** [comment]
@yajo thanks a lot for the in depth analysis! In our own runbot setup we are only testing a subset of the oca modules from this repository (which does not include mail_autosubscribe). Guess that explains why we stumbled over this while not seeing the import error.

**Comment by yajo** [comment]
The fix is in https://github.com/OCA/social/pull/934, so once that one's merged, rebase this PR please.

**Comment by dhx** [comment]
> The fix is in #934, so once that one's merged, rebase this PR please.

@yajo rebase is pushed

**Review by yajo — APPROVED** [review]
I can see now in the logs the test running fine.

**Pull request merged**
